alignak/http/client.py

```diff
@@ -156,7 +157,12 @@ def get(self, path, args={}, wait='short'):
         uri = self.make_uri(path)
         timeout = self.make_timeout(wait)
         try:
-            rsp = self._requests_con.get(uri, params=args, timeout=timeout, verify=self.strong_ssl)
+            headers = {
+                'content-type': 'application/zlib',
```

Comment:
We may have talked about it but are you sure it's always in zlib format? Maybe  this statement force the zlib format?